homework/quang_tu/subarray-sums-divisible-by-k.cpp

```diff
@@ -0,0 +1,26 @@
+class Solution {
+public:
+    int subarraysDivByK(vector<int>& nums, int k) {
+        unordered_map<int,int> hashMap;
+        hashMap[0] = 1;
+
+        int sum = 0, mod, ans = 0;
+
+        for (int i = 0; i < nums.size(); ++i) {
+            sum += nums[i];
+
+            mod = sum%k;
```

Comment:
nitpick: better use consistent style. Consider applying https://google.github.io/styleguide/cppguide.html. If you use vscode or jetbrain, I believe they have some shortcut to format your code automatically